sre_maturity.md

```diff
@@ -0,0 +1,115 @@
+
+### Characteristics
+
+#### People and Culture
+
+You have a strong SRE voice in your team, someone who advocates for reliability features.
+They establish people focused mantras like "It's OK to fail", "Hope is not a strategy" and "Trust by default & extend on it".
+Communication is important in your team and you hire diverse people based on soft skills & leadership rather than specific technologies.
+
+At this phase, having buy-in from executive level leadership is essential.
+Without this it will be very difficult to get their support later, and eventually their engagement.
+
+The product manager for you service should have a good understanding of SRE.
+They should help you advocate for service features that make the service more reliable.
+
+#### Observability
+
+The foundations for monitoring and observing your service are in place.
+Being able to measure something meaningful about your service is important, and supplements any feedback you get from initial users.
+A common way to do this is via metrics exposed by your service.
+
+You have a simple Service Level Objective (SLO) for you service to start out with.
+It doesn't have to be completely representative of what users expect from your service, but it forms the building blocks of an SLO review process and improving on them in time.
+Starting with a lower SLO than where you want to get to sets expectations early.
+
+#### “Admin” access to the service
+
+Your team has the level of access required to fix production problems.
+If not, you can’t expect them to debug and fix problems in a timely manner.
+Being overly controlling of access shouldn’t be a priority at this phase.
+That can come later with things like audited access, permissions escalation & tiered access as you mature the debugging experience of your service.
+You should trust your team and assume positive intent.
+
+#### On Call, Escalations and Incidents
+
+You have a team with an on call schedule.
+It doesn’t have to be 24/7. It doesn’t have to be automated.
+But there must be someone responsible to respond when things go bad.
+Setting expectations at this early stage with a reduced SLO means you can tailor the schedule to more reasonable times for your team.
+
+Following on from this, you have an escalation process. A way to be notified of problems. 
+It should be possible to raise a problem with the SRE team and the people that build the service, like engineers, QE, writers.
+This can be a mailing list, slack channel, issue system, whatever.
+Formalise it and share it.
+
+If things go really bad, you write up an RCA.
+You should be starting to build up a library of RCAs that can feed into doing more proactive things as you mature e.g. Incident rehearsals.
+Each RCA includes a 'blameless' post mortem session with the people involved while the incident is still fresh in their head.
+This post mortem feeds important reliability improvements back to the team.
```

Comment:
Add emphasis here that the value of post mortems is in the changes they recommend. Lack of follow-up here is all too common and management should ensure follow-up.

Reply:
I should say ... management should ensure high priority of RCA action items and track follow through

Reply:
```suggestion
Each post mortem feeds *extremely valuable* reliability improvements back to the team.
Your management team should ensure high priority of these actions & follow through on them.
```
@jeremyeder Added emphasis and wording about prioritising & following through.